addons/hitreactions/functions/fnc_throwWeapon.sqf

```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+/*
+ * Author: commy2
+ * Makes the unit throw their currently selected weapon. Unit must be local.
+ *
+ * Arguments:
+ * 0: unit <OBJECT>
+ *
+ * Return Value:
+ * Weapon Holder
+ *
+ * Example:
+ * player call ACE_hitreactions_fnc_throwWeapon
+ *
+ * Public: No
+ */
+
+#define THROW_VELOCITY 1.5
+#define THROW_TORQUE 0.2
+
+params ["_unit"];
+private _weapon = currentWeapon _unit;
+if (_weapon isEqualTo "") exitWith {objNull};
+
+private _data = weaponsItems _unit select {_x select 0 == _weapon} select 0;
+
+private _holder = createVehicle ["WeaponHolderSimulated", [0,0,0], [], 0, "CAN_COLLIDE"];
+_holder addWeaponWithAttachmentsCargoGlobal [_data, 1];
```

Comment:
:smiley:

Reply:
It's a really fucking great command, dude. Look at L25. That's all you need to get the relevant info. Splendid job.

Reply:
Well my command draft would've been slightly different format than weaponsItems, but whoever copy-pasted it fixed that, dunno who to thank for that.

Reply:
That's not something you admit.